### DISCUSSION TITLE
Model settings in Companion not matching Model settings in radio/on SD.

### DISCUSSION THREAD

**Discussion by dcan999** [EdgeTX/edgetx#5345] [General]
I am just learning how to use companion so I might be doing something wrong. I plug in my Zorro, which is running 2.10.2, select SD Storage mode, then open Companion 2.8.3, select Zorro profile. 

I then select "read models and settings", and it opens that screen. 

When I open a model, some of the settings don't match what's actually in the radio/SD.

 Example, all of the Special functions I have on the radio are Play Tracks for switch positions. 

On the radio, they all have no repeat. In Companion, they all show as have 1s repeat. 
I have switch warnings set up for 5 switches, Companion shows none set up. 

Only some models do this, other seem to match. 

Am I doing something wrong?

**Comment by pfeerick** [comment]
Yes. You should **never** use an older version of Companion than the
firmware you are running. If you are Running 2.10.2 on your radio, you
should be Using Companion 2.10.2 or later to configure it. Otherwise
settings that have changed their structure, and settings that didn't exist
in the earlier Companion, **will** be mangled.

**Reply by dcan999** [reply]
Thanks, I didn't know the versions were matching. I actually did download 2.10 on my PC but it didn't make a shortcut so I just thought the 2.8 shortcut was the one I downloaded. I just opened 2.10 and all settings match! derp.
